Size your shards - Expand "Number of documents in the shard cannot exceed" section (#4723)
elastic/docs-content: deploy-manage/production-guidance/optimize-performance/size-shards.md

This limit is much higher than the [recommended maximum document count](#shard-size-recommendation) of approximately 200M documents per shard.

> If you encounter this problem, try to mitigate it by using the [Force Merge API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-forcemerge) - run force-merge during off-peak hours and preferably on indices that are no longer receiving writes - to merge away some deleted docs. For example:

Reviewer: ```suggestion
If you encounter this problem, try to mitigate it if possible during off-peak hours by using the [Force Merge API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-forcemerge) to expunge away some deleted documents. For example:
```
